packages/app/client/src/ui/pages/Account/components/Dashboard.js

```diff
@@ -0,0 +1,222 @@
+  },
+  editButton: {
+    marginRight: '5%',
+    width: '107px',
+    height: '42px'
+  },
+  viewProject: {
+    width: '139px',
+    height: '42px'
+  }
+}))
+
+function Dashboard(props) {
+  const { handleChange } = props
+  const classes = useStyles()
+  const compareByDate = (a, b) => {
+    if (a.date > b.date) return -1
+    if (a.date < b.date) return 1
+    return 0
+  }
+  return (
+    <>
+      <Typography style={{ marginBottom: '2%', fontSize: '34px' }} variant="h3">
+        Hello, {props.name}
+      </Typography>
+      <Grid item container xs={12}>
+        <Grid item xs={12} sm={12} md={6}>
+          <BorderedInfo size="normal">
+            <Typography style={{ paddingTop: '4%' }} variant="subtitle1">MY STATS</Typography>
+            <div className={classes.boxes}>
+              <div style={{ textAlign: 'center' }}>
+                <div className={classes.bordered}>
+                  <div className={classes.topText}>
+                    <LikeButton liked onlyIcon disabled />
+                    <div>Likes</div>
+                  </div>
+                  <div style={{ borderBottom: '2.2px solid #15B54A' }} />
+                  <div className={classes.count}>23</div>
+                </div>
+                <Button className={classes.buttons} color="primary">
+                  Redeem likes
+                </Button>
+              </div>
+              <div style={{ textAlign: 'center' }}>
+                <div>
+                  <div className={classes.bordered}>
+                    <div className={classes.topText}>
+                      <img style={{ marginRight: '5%' }} src={Badge} />
+                      <div>Badges</div>
+                    </div>
+                    <div style={{ borderBottom: '2.2px solid #15B54A' }} />
+                    <div className={classes.count}>
+                      {props.badges &&
+                        Object.keys(props.badges).filter(
+                          key => props.badges[key]
+                        ).length}
+                      /{Object.keys(props.badges).length}
+                    </div>
+                  </div>
+                </div>
+                <Button
+                  onClick={e => handleChange(e, 6)}
+                  className={classes.buttons}
+                  color="primary"
+                >
+                  View Badges
+                </Button>
+              </div>
+            </div>
+          </BorderedInfo>
+        </Grid>
+        <Grid item xs={12} sm={12} md={6}>
+          <BorderedInfo size="normal">
+            <div className={classes.header}>
+              <Typography variant="subtitle1">MY LATEST PROJECT</Typography>
+              <Button
+                className={classes.seeAllButton}
+                onClick={e => handleChange(e, 4)}
+              >
+                see all >
+              </Button>
+            </div>
+            <div>
+              <div style={{ width: '100%', height: '150px' }}>
```

Comment:
You could use the Box component from Material UI for this purpose and avoid the inline styling